internal/ui/peerpage.go

```diff
@@ -303,19 +303,27 @@ func peerSubtitle(peer *ipnstate.PeerStatus) string {
 	return ""
 }
 
-func peerIcon(peer *ipnstate.PeerStatus) string {
+var (
+	peerIconExitNodeOffline = gio.NewThemedIconFromNames([]string{"network-vpn-acquiring-symbolic"})
```

Comment:
Should this also include `"security-high-symbolic"` as a fallback too? This is the one which was actually missing.

Reply:
Yep. I need to find a good way to come up with a list that'll both work and look nice across different desktops. I have KDE installed, so I just need to switch into that when I get a chance and experiment a bit.

For this particular one, I'm not sure what fallback makes more sense. The current icon is supposed to both indicate that it's being used as an exit node and that it's disconnected, and none of the fallbacks do both of those things at the same time well.

I think it being disconnected is more important for the icon to show, as it being an exit node is also indicated in the row's subtitle.

Reply:
Unfortunately, it doesn't look like there's a way to fall back from a theme that isn't installed, just fallbacks for icons missing within a theme. I'm not sure what the behavior is if I try to force a theme that isn't installed at all. It might be dependent on the environment.

I'll experiment with forcing Adwaita and see if I can find icons in that theme that I like.